NuGet-Packages.md

```diff
@@ -0,0 +1,51 @@
+NuGet packages are a convenient way to provide developers with official builds of our project while enabling
+
+See [our plans for NuGet (#85)](https://github.com/Microsoft/ChakraCore/issues/85).
+
+# NuGet Feed (Stable Builds)
+
+_Coming soon! (Check back when we finalize the `release/1.2` milestone.)_
+
+# MyGet Feed (Preview Builds)
+
+We publish preview builds of our NuGet package at the
+[Microsoft.ChakraCore](https://www.myget.org/feed/chakracore-preview/package/nuget/Microsoft.ChakraCore)
+MyGet Feed.
+
+Currently, packages on the MyGet feed are built from the `release/1.2` branch.
+We have plans to expand this to preview packages built from any branches under active development
+(such as `master`). Builds are published regularly at a rate consistent with changes to the branch
```

Comment:
I think the verbage related to rates of change in branches might be a bit confusing as well. It's hard to put a publication schedule on a branch like release/1.2 since it really depends on when it makes sense to update the package.